torch_xla/experimental/tpu.py

```diff
@@ -81,11 +81,26 @@ def task_id() -> Optional[int]:
   return xu.getenv_as(xenv.CLOUD_TPU_TASK_ID, int)
 
 
+def build_tpu_env_from_vars() -> Dict[str, str]:
+  metadata = defaultdict(str)
```

Comment:
Why use `defaultdict`?

Reply:
The idea was to prevent crashes when consumers of `tpu_env` query using a key that would be in GCE metadata but is missing from the environment variables

Reply:
Replaced with `dict()`